test/e2e/demos_test.go

```diff
@@ -23,26 +23,51 @@ func skipIfNotKubernetes() {
 	}
 }
 
-var _ = Describe("Introduction: Configuration", func() {
+var _ = Describe("Introduction", func() {
```

Comment:
Maybe it's super stupid idea, but would be nice to have a test that fails if docs has being updated but test are not.

Reply:
The tests are using the same examples as docs. And docs have a note saying "update the example if you update the .md". I hope that is enough, we just need to keep in mind to update tests. I don't consider it worthy an advanced mechanism to check diffs in doc. KISS